cocos/asset/assets/texture-cube.ts

```diff
@@ -362,6 +416,7 @@ export class TextureCube extends SimpleTexture {
      */
     public destroy () {
         this._mipmaps = [];
+        this._generatedMipmaps = [];
```

Comment:
Why you need an extra variable？

Reply:
It's necessary, `updateMipmaps` needs the generated mipmaps to upload the selected data. But don't worry, the generated mipmaps are just temp views that are excluded from serialization.